src/Widgets/DiskBar.vala

```diff
@@ -207,6 +213,10 @@ public class Installer.DiskBar: Gtk.Grid {
     }
 
     internal class FillRound : Gtk.Widget {
+        internal FillRound () {
```

Comment:
Is this empty method needed?

Reply:
No idea. At the time of writing this code, I simply copied the pre-existing storage bar Vala code to use as a template for implementing the disk bar, which had this empty method.